internal/ncproxystore/store.go

```diff
@@ -0,0 +1,279 @@
+package ncproxystore
```

Comment:
Instead of internal/ncproxystore and internal/ncproxynetworking, is it possible to have an internal/ncproxy pkg and networking and store sub packages within?